api/rc3.py

```diff
@@ -98,7 +99,7 @@
 
 	dev_range = rq.args.pop("range", None)
 	filters = rq.args
-	db_proxy = await getDeviceProxy('sys/database/2')
+	db_proxy = await getDeviceProxy(db.get_name())
```

Comment:
This is perfect. You resolve both issue, a default database and still able to do async request.